src/sites/link/sylnk.net.js

```diff
@@ -237,6 +237,17 @@ _.register({
   },
 });
 
+_.register({
+  rule: {
+    host: /^hikarinoakariost\.info$/,
+  },
```

Comment:
This site seems use base64 to encode the redirected URL.
So maybe add `query: /^\?(.+)$/`, and use `atob` for `query[1]`.

Reply:
Completely missed out on that..